llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -3606,6 +3613,32 @@ bool SPIRVInstructionSelector::selectSpvThreadId(Register ResVReg,
   return Result && MIB.constrainAllUses(TII, TRI, RBI);
 }
 
+bool SPIRVInstructionSelector::selectSpvThreadId(Register ResVReg,
+                                                 const SPIRVType *ResType,
+                                                 MachineInstr &I) const {
+  // DX intrinsic: @llvm.dx.thread.id(i32)
+  // ID  Name      Description
+  // 93  ThreadId  reads the thread ID
+  //
+  // In SPIR-V, llvm.dx.thread.id maps to a `GlobalInvocationId` builtin
+  // variable
```

Comment:
This comment block seems odd, since we should be translating a SPIRV intrinsic: `@llvm.spv.thread.id`, not a DX intrinsic.  Same for the one in `selectSpvGroupThreadId`.

Reply:
@tex3d Done. The comment was updated in the latest commit of the PR.